joomla/joomla-cms
# [4.2] Enable/disable 'Upload&Install' button on the basis of checkbox value

Pull Request for Issue #36274 and #35055 .

### Summary of Changes
Added disabled state to the "UploadAndInstall" button.

### Testing Instructions

1.System-->Update-->Joomla.
2. Click on "Update your site by manually uploading the update package".
3. Button and checkbox are disabled initially.
4. Upload a file greater than 40 MB, checkbox and button are still disabled.
5. upload a file less than 40 MB, the checkbox is enabled. Check in the box, the button gets enabled.
6. again upload a file greater than 40 MB, checkbox and button get disabled again.

### Actual result BEFORE applying this Pull Request
![1](https://user-images.githubusercontent.com/54266411/151229601-a0512cda-7774-4c2f-89f2-a14e8163ea0a.png)



### Expected result AFTER applying this Pull Request


https://user-images.githubusercontent.com/54266411/151861745-104601f0-9a7c-4ef7-8479-108c6b4d1f2d.mp4
